### PR TITLE
FE-2: added team members page

### PR DESCRIPTION
**Pull request by mnyx7**

### CODE REVIEW

File: response.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        "email": "minaya.yagubova@gmail.com",
+        "linkedin": "https://www.linkedin.com/in/mnyx7/",
+        "github": "https://github.com/mnyx7",
+        "position": "Front-End",
+        "photo": "photo/minaya"
+    },
+    {
+        "firstName": "Mahammad",
+        "lastName": "Rahimov",
+        "email": "m.ragimov09@gmail.com",
```
Comment (NigarMovsumova) [inline response.json]: ?

File: response.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        "firstName": "Ali",
+        "lastName": "Huseynov",
+        "email": "huseynov_ali@outlook.com",
+        "linkedin": "https://www.linkedin.com/mwlite/in/ali-huseynov-a3198b160",
+        "github": "https://github.com/alihsoff",
+        "position": "Back-End",
+        "photo": "photo/ali"
+    },
+    {
+        "firstName": "new",
```
Comment (NigarMovsumova) [inline response.json]: No information, if you do not need it, remove it.

File: team.css [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+header .navTools .languages a:hover {
+	color: #117a65;
+	font-weight: 900;
+	font-size: 17px;
+}
+
+
+
+
+
```
Comment (NigarMovsumova) [inline team.css]: ?

File: index.htm [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+						<a href="#">EN</a>
+						<a href="#">RU</a>
+					</div>
+				</div>
+				<div class="tool">
+					<i class="open-modal-btn fas fa-user-circle"></i>
+				</div>
+			</ul>
+		</nav>
+	</header>
```
Comment (NigarMovsumova) [inline index.htm]: make this a reusable component to reduce duplicated code.

File: script.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+window.onload = function () {
+
+    fetch('./response.json')
+        .then(res => res.json())
+        .then(json => {
+            var  team = document.getElementsByClassName('team')[0]
```
Comment (NigarMovsumova) [inline script.js]: You do not use "team" as a className, you use it as an id anywhere in your code.

File: script.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+window.onload = function () {
+
+    fetch('./response.json')
```
Comment (NigarMovsumova) [inline script.js]: declare a function for this case.

File: script.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        records.forEach(record => {
+            
+
+            team.innerHTML += `
+            <div class="member">
+                    <div class="face photo">
+                    <img src="${record.photo}" alt="${record.position}" />
+                        <h1 class="name">${record.firstName} ${record.lastName}</h1>
+                        <p class="position">${record.position}</p>
+                        <div class="m-icons">
```
Comment (NigarMovsumova) [inline script.js]: make class name more readable, another developers reading your code should understand what any variable, function name, class or id do in your code just by reading the code.

File: script.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+    fetch('./response.json')
+        .then(res => res.json())
+        .then(json => {
+            var  team = document.getElementsByClassName('team')[0]
+            renderMember(json)
+        })
+
+
+        const renderMember = records => {
```
Comment (NigarMovsumova) [inline script.js]: why records, record ? Change the name of it to make it more readable.

File: script.js [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+        records.forEach(record => {
+            
+
+            team.innerHTML += `
+            <div class="member">
+                    <div class="face photo">
+                    <img src="${record.photo}" alt="${record.position}" />
```
Comment (NigarMovsumova) [inline script.js]: alt should be name and surname, position is not unique for each team member.

File: team.css [hunk 10]
```diff
@@ -0,0 +1,255 @@
...
+header nav {
+	position: fixed;
+	width: 100%;
+	top: 0;
+	height: 70px;
+	background-color: #48c9b0;
+	display: flex;
+    justify-content: space-between;
+    z-index: 1;
+}
```
Comment (NigarMovsumova) [inline team.css]: indentation

File: index.htm [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+</body>
+
+</html>
+
+
+<script src="https://kit.fontawesome.com/a076d05399.js"></script>
+<script src="script.js"></script>
+</body>
+
+</html>
```
Comment (NigarMovsumova) [inline index.htm]: Why body html closing tags are mentioned twice ? Please, be more careful.

File: index.htm [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+			<i class="fab fa-instagram"></i>
+			<i class="fab fa-pinterest-p"></i>
+			<i class="fab fa-google-plus-g"></i>
+			<i class="fab fa-github"></i>
+		</div>
+		<section class="end">
+			<p>Copyright&copy; 2020. E-learning </p>
+			<p>Bütün hüquqlar qorunur</p>
+		</section>
+	</footer>
```
Comment (NigarMovsumova) [inline index.htm]: reusable component.

File: reset.css [hunk 13]
```diff
@@ -0,0 +1,53 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
```
Comment (NigarMovsumova) [inline reset.css]: remove comments.

File: script.js [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+                        <h1 class="name">${record.firstName} ${record.lastName}</h1>
+                        <p class="position">${record.position}</p>
+                        <div class="m-icons">
+                            <a href="mailto:${record.email}" class="fas fa-envelope-square mymail"></a>
+                            <a href="${record.linkedin}" class="fab fa-linkedin mylink"></a>
+                            <a href="${record.github}" class="fab fa-github-square mygit"></a>
+                        </div>
+                    </div>
+                    <div class="face info">
```
Comment (NigarMovsumova) [inline script.js]: face info ?

File: team.css [hunk 15]
```diff
@@ -0,0 +1,255 @@
...
+.info{
+    color: transparent;
+    transform: translateY(-50px);
+}
+.member:hover .info{
+    box-shadow: 0 4px 8px 0 #48C9B0;
+    color: #fff;
+    transform: translateY(0);
+    background-color: #48C9B0 !important;
```
Comment (NigarMovsumova) [inline team.css]: do not use important. Replace it.

File: response.json (hunk 1 above)
Comment (mnyx7) [reply response.json]: couldn't  understand

File: index.htm (hunk 4 above)
Comment (mnyx7) [reply index.htm]: It's not mine it's Mahammad's

File: team.css (hunk 10 above)
Comment (mnyx7) [reply team.css]: It's not mine it's Mahammad's

File: index.htm (hunk 12 above)
Comment (mnyx7) [reply index.htm]: It's not mine it's Mahammad's

File: reset.css (hunk 13 above)
Comment (mnyx7) [reply reset.css]: It's not mine it's Mahammad's

File: team.css (hunk 15 above)
Comment (mnyx7) [reply team.css]: It's not mine it's Mahammad's

**Pull request closed without merging**